actions/merkleroots/bhs_error_response.go

```diff
@@ -0,0 +1,46 @@
+package merkleroots
+
+import (
+	"encoding/json"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/gin-gonic/gin"
+	"github.com/go-resty/resty/v2"
+	"github.com/rs/zerolog"
+)
+
+type bHSErrorResponse struct {
```

Comment:
Lowercased "b" ?

Reply:
It's only used in this file and for parsing the response error to later parse it to spverror. I think it's unnecessary to make it exported, but I can change it if you'd like

Reply:
OK. So please change it to less weird form 🙈 
```suggestion
type bhsErrorResponse struct {
```